app/src/main/res/layout/set_profile_4_2.xml

```diff
@@ -25,6 +25,7 @@
         android:layout_height="50dp"
         android:contentDescription="@string/please_specify_gender"
         android:ems="10"
+        android:hint="@string/more_hint"
```

Comment:
does the hint get covered by the contentDescription ? if it is the case remove one of the two

Reply:
hint is the one that appears greyed out before you start to type something. contentDescription was for Accessibility if I'm not mistaken.